src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -20,15 +20,19 @@
         private const string CommandLineEventName = "CommandLineInformation";
         private const string AssemblyReferencesEventName = "AssemblyReferencesInformation";
 
-        private readonly int ChunkSize;
-        private readonly string sha256;
+        private string Sha256 => context?.Hashes?.Sha256;
```

Comment:
>context?.Hashes?.Sha256 [](http://example.com/codeflow?start=33&length=23)

since we changed how hashes are generated, we should make a reference for the context so we will have the correct value of the hash when needed.